pkg/sourcestest/sourcestest.go

```diff
@@ -0,0 +1,61 @@
+package sourcestest
```

Comment:
I wasn't sure where to put these so I made a `sourcestest` package akin to [iotest](https://pkg.go.dev/testing/iotest). Open to suggestions!

Reply:
Oh I have a test implementation https://github.com/trufflesecurity/trufflehog/pull/1877 but open to this too.